lib/rules/new-line-between-multi-line-property.js

```diff
@@ -0,0 +1,90 @@
+/**
+ * @fileoverview Enforce new lines between multi-line properties in Vue components.
+ * @author IWANABETHATGUY
+ */
+'use strict'
+const _ = require('lodash')
+// ------------------------------------------------------------------------------
+// Rule Definition
+// ------------------------------------------------------------------------------
+
+module.exports = {
+  meta: {
+    type: 'layout',
+    docs: {
+      description:
+        'enforce new lines between multi-line properties in Vue components',
+      categories: undefined,
+      url:
+        'https://eslint.vuejs.org/rules/new-line-between-multi-line-property.html'
+    },
+    fixable: 'code', // or "code" or "whitespace"
+    schema: [
+      {
+        type: 'object',
+        properties: {
+          // number of line you want to insert after multi-line property
+          'insert-line': {
+            type: 'number',
+            minimum: 1
+          },
+          'min-line-of-multiline-property': {
+            type: 'number',
+            minimum: 2
+          }
+        }
+      }
+    ]
+  },
+
+  create(context) {
+    const insertLine = _.get(context, ['options', '0', 'insert-line'], 1)
```

Comment:
Forcing two or more blank lines conflicts with `no-multiple-empty-lines` rule, so do not allow the number of lines to be set. 
Instead just check if there is more than one blank line.